src/index.html

```diff
@@ -8,6 +8,25 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+
+    <!-- <div class="container">
+      <ul>
+        <li class="stars stars--0"></li>
+        <li class="stars stars--0"></li>
+        <li class="stars stars--0"></li>
+        <li class="stars stars--0"></li>
+        <li class="stars stars--0"></li>
+      </ul>
+    </div> -->
+
+    <div class="container">
+      <div class="stars stars--0"></div>
+      <div class="stars stars--1"></div>
+      <div class="stars stars--2"></div>
+      <div class="stars stars--3"></div>
+      <div class="stars stars--4"></div>
+      <div class="stars stars--5"></div>
+    </div>
```

Comment:
```suggestion
    <div class="stars stars--0">
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
    </div>
```
Вот так должны выглядеть твои блоки, Каждый следующий должен отличаться классом stars--1, stars--2, stars--3 и так далее. Число `--3` отвечает за количество активных звездочек